koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -482,6 +288,7 @@ def handler(self, src, target_info, arch, output_template, scratch=False,
             'target': target_info['name'],
             'architecture': arch,
             'yum_repourls': yum_repourls,
+            'scratch': scratch
```

Comment:
Tiny nit: could add a trailing comma at the end, just so any future additions are easy. Won't hold off merging for this though.